website/docs/components/pagination/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,44 @@
+## Usage
+
+### When to use
+
+- To break down large content into pages. Usually paired with tables, but works with other types of page content.
+
+### When not to use
+
+- As a navigation control for a flow or to pair with a stepper, i.e. for a guide, tutorial, setup flow.
+- As a controller to switch between multiple views. Use [Tabs](/components/tabs/) instead.
+
+### Numbered vs Compact
```

Comment:
```suggestion
### Numbered vs compact
```

Reply:
I'm hesitant about this one because we refer to to a variant name, so I feel like it's correct in this case, but I'm not sure. I can't find a similar example on another component. @MelSumner @heatherlarsen do you have an opinion on this?

Reply:
Feel free to leave as is if you disagree. We could always update later if needed

Reply:
Hmm, I suppose this is a question about whether or not we consider variant and property names as proper nouns. If we consider them proper nouns, then they should be capitalized. If not, then they should be lowercase. I'm leaning toward it being capitalized, but very curious about what others think.

**Reference**
"In title case, major words are capitalized, and most minor words are lowercase. In sentence case, most major and minor words are lowercase ([proper nouns](https://apastyle.apa.org/style-grammar-guidelines/capitalization/proper-nouns) are an exception in that they are always capitalized)." - [APA style](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)